protocols/imap/src/main/java/org/apache/james/imap/decode/parser/CreateCommandParser.java

```diff
@@ -46,7 +46,6 @@ public CreateCommandParser() {
      */
     protected ImapMessage decode(ImapCommand command, ImapRequestLineReader request, String tag, ImapSession session) throws DecodingException {
         String mailboxName = request.mailbox();
-
```

Comment:
It's because my IDE which auto remove while space. Could we accept this because it's useful I think?